recipes/prep.rb

```diff
@@ -0,0 +1,2 @@
+# this is required to run git dependencies via inspec
+package 'git'
```

Comment:
This recipe seems out of place if only for the purpose of being used during the testing phase in kitchen of this cookbook.  In my opinion, a better approach is to install it via a fixture cookbook.